docs/run_test/transformers.rst

```diff
@@ -288,6 +292,12 @@ type: bool | Default: true
 
 Whether to create a new deployment. If true, creates a new VM deployment. If false, uses existing VMs in the specified resource_group_name.
 
+source_address_prefixes
```

Comment:
Please also add comments to explain if those values are not provided, the platform configuration will be used for the transformer. When the vm of transformer has different network infra, it can be overwritten here.